lib/src/binding_coap/coap_client.dart

```diff
@@ -192,6 +192,69 @@ class CoapClient extends ProtocolClient {
     return response.content;
   }
 
+  /// Obtains an ACE creation hint serialized as a [List] of [int] from a
+  /// [Form].
+  ///
+  /// Returns `null` if no `ACESecurityScheme` is defined.
+  Future<AuthServerRequestCreationHint?> _obtainAceCreationHintFromForm(
+    Form? form,
+  ) async {
+    if (form == null) {
+      return null;
+    }
+
+    final aceSecuritySchemes = form.aceSecuritySchemes;
+
+    if (aceSecuritySchemes.isEmpty) {
+      return null;
+    }
+
+    // TODO: Does a boolean cnonce parameter make sense here?
+    final aceSecurityScheme = aceSecuritySchemes.first;
+
+    AuthServerRequestCreationHint? creationHint;
+
+    if (aceSecurityScheme.cnonce == true) {
```

Comment:
```suggestion
    if (aceSecurityScheme.cnonce) {
```

Reply:
`cnonce` is actually nullable, therefore the comparison with `null`. I suppose this should make things more clear?

```suggestion
    if (aceSecurityScheme.cnonce ?? false) {
```

Reply:
Applied the change in 9f2f3e0fed65892c652e609ee0175b07bff6070e

Reply:
Ah,-my-bad,I-didn't-notice.The-solution-with-`??`-is-a-good-idea.(Still-have-that-spacebar-problem...)